hawkular-alerts-rest-tests/pom.xml

```diff
@@ -165,8 +165,9 @@
 
   <profiles>
 
+    <!-- This profile requires a running server of hawkular or hawkular-alerts/standalone -->
     <profile>
-      <id>rest-only</id>
+      <id>itest-tests</id>
```

Comment:
Please change to -Pitest-external to make more verbose that this option requires the external server to be running.